python/setup.py

```diff
@@ -13,6 +13,7 @@
             'argparse-formatter>=1.4',
             'colorama>=0.4.6',
```

Comment:
Make sure to rebase your other PR where you change the `colorama` version here

Reply:
Yep thanks for the reminder